code/modules/antagonists/cult/cult_items.dm

```diff
@@ -252,28 +358,64 @@ Striking a noncultist, however, will tear their flesh."}
 	name = "[path_spells[SWORD_PREFIX]] [name]"
 
 
-	// Granting the path spells. The sword spirit gains it outright, while it's just instanced for wielders to be added on pickup.
+	// Creating the path spells.
+	// The sword is created bound - so we do not grant it the spells just yet, but we still create and store them.
 
 	if(sword_spells)
 		for(var/datum/action/cooldown/spell/sword_spell as anything in sword_spells)
-			sword_spell = new sword_spell(trapped_entity)
-			sword_spell?.Grant(trapped_entity)
-			sword_spell?.overlay_icon_state = "bg_cult_border" // for flavor, and also helps distinguish
-
+			var/datum/action/cooldown/spell/instanced_spell = new sword_spell(trapped_entity)
+			LAZYADD(path_sword_actions, instanced_spell)
+			instanced_spell.overlay_icon_state = "bg_cult_border" // for flavor, and also helps distinguish
 
 	if(wielder_spells)
 		for(var/datum/action/cooldown/spell/wielder_spell as anything in wielder_spells)
-			path_wielder_action = new wielder_spell(src)
-			wielder_spell?.overlay_icon_state = "bg_cult_border"
+			var/datum/action/cooldown/spell/instanced_spell = new wielder_spell(trapped_entity)
+			LAZYADD(path_wielder_actions, new wielder_spell(src))
+			instanced_spell.overlay_icon_state = "bg_cult_border"
```

Comment:
Two spells are created here, and one doesn't get a cult background, is that intentional?

Reply:
oop